cdap-storage-ext-spanner/src/main/java/io/cdap/cdap/storage/spanner/SpannerStructuredTableAdmin.java

```diff
@@ -372,6 +401,28 @@ private void createTable(StructuredTableSpecification spec) throws IOException {
     }
   }
 
+  @Override
+  public void batchCommit() throws IOException {
+    if (ddlStatements.isEmpty()) {
+      LOG.debug("No ddl statements to execute");
+      return;
+    }
+    try {
+      Uninterruptibles.getUninterruptibly(
```

Comment:
This is the same logic as createTable. Create a new private method that gets. List<String> ddls and execute them. You can then call it inside createTable and batchCommit

Reply:
Done.